### DISCUSSION TITLE
enter unicode character in wxmaxima

### DISCUSSION THREAD

**Discussion by jfmaurel** [wxMaxima-developers/wxmaxima#1530] [General]
I would like to enter unicode character with hexadecimal code "2227" decimal "8743"

I tried the following as written in the documentation but it does not work for me:
'If a special symbol isn’t in the list it is possible to input arbitrary Unicode characters by pressing ESC [number of the character (hexadecimal)] ESC.'

Could you please tell what I am missing here?

Regards

PS:	build_info(version="branch_5_44_base_201_gf4e9e33e6",timestamp="2020-12-21 00:04:33",host="x86_64-w64-mingw32",lisp_name="SBCL",lisp_version="2.0.0",maxima_userdir="C:/Users/mlamo/maxima",maxima_tempdir="C:/Users/mlamo/AppData/Local/Temp",maxima_objdir="C:/Users/mlamo/maxima/binary/branch_5_44_base_201_gf4e9e33e6/sbcl/2_0_0",maxima_frontend="wxMaxima",maxima_frontend_version="20.11.1_MSW_OpenMP201511+Locks")build_info(version="branch_5_44_base_201_gf4e9e33e6",timestamp="2020-12-21 00:04:33",host="x86_64-w64-mingw32",lisp_name="SBCL",lisp_version="2.0.0",maxima_userdir="C:/Users/mlamo/maxima",maxima_tempdir="C:/Users/mlamo/AppData/Local/Temp",maxima_objdir="C:/Users/mlamo/maxima/binary/branch_5_44_base_201_gf4e9e33e6/sbcl/2_0_0",maxima_frontend="wxMaxima",maxima_frontend_version="20.11.1_MSW_OpenMP201511+Locks")(version="branch_5_44_base_201_gf4e9e33e6",timestamp="2020-12-21 00:04:33",host="x86_64-w64-mingw32",lisp_name="SBCL",lisp_version="2.0.0",maxima_userdir="C:/Users/mlamo/maxima",maxima_tempdir="C:/Users/mlamo/AppData/Local/Temp",maxima_objdir="C:/Users/mlamo/maxima/binary/branch_5_44_base_201_gf4e9e33e6/sbcl/2_0_0",maxima_frontend="wxMaxima",maxima_frontend_version="20.11.1_MSW_OpenMP201511+Locks")

**Comment by Parlazo** [comment]
maybe its a problem with the font in the text display ... may some how the unicode panel can show more symbols than the text block can write/display
https://github.com/wxMaxima-developers/wxmaxima/discussions/1516
not every font has every symbol .. may you need to find a font which contains all symbols you need

**Reply by jfmaurel** [reply]
Hi,

Thank you for your answer.
A unicode font is selected in configure for configure/style documentation+code font and math font
Use unicode math symbol if available is selected.
This does not seem to change the behaviour.

Is there any other settings I would be missing?

Regards

**Reply by Parlazo** [reply]
https://unicode.flopp.net/c/2227 logical AND
i copy the character and paste it into wxMaxima and i can see it in the text and math blocks.
i use in wxMaxima under the maxima style settings for docu+code "Courier New" and as math font "Lucida Sans Unicode" and under this two main font settings are a lot of detail settings where i use "Lucida Sans Unicode" and color as i like.
I think its a problem with some fonts ... you have to search a font which supports most symbols you need... some uniode fonts have more or less math symbols. i did try some and many did miss some symbols and even Lucida Sans Unicode is missing some but many others miss much more symbols... to me Lucida Sans Unicode seems the most complete font at the moment and it is not but looks kind of monospaced
but in the math blocks i use Courier New and the symbol for logical AND does appear to
i think the default in wxMaxima is some font with name Linux Symbols or some think like that ... maybe you have to find a better font
unicode font does not mean it contains all possible unicode symbols... you may search for a more complete unicode font which contains the symbols you need in your project

**Reply by jfmaurel** [reply]
Hi,

You are right it works with the symbol 'and'.

Thank you.
